src/Pages/Playlist.js

```diff
@@ -0,0 +1,90 @@
+import { useLibraryContext } from "../Context/LibraryContext";
+import { VideoCard } from "../Components/VideoCard";
+import { NavBar } from "../Components/NavBar";
+import { useState } from "react";
+import { Footer } from "../Components/Footer";
+import { Toast } from "../Components/Toast";
+import { useToggleContext } from "../Context/ToggleContext";
+
+export const Playlist = () => {
+  const { state, dispatch } = useLibraryContext();
+  const [reRender, setReRender] = useState(false);
```

Comment:
Could not understand the purpose of manual rerender

Reply:
the dispatch does not re-render, as there are two mapping functions involved (my guess, from what I read). So, I had to do a manual re-render.

Reply:
Dispatch does not cause a render, but the context will render if the state value has changed. But if it has not rendered manual rerender is fine. We can have a look at it later.

Reply:
Sure!